### DISCUSSION TITLE
column name order in select giving exception error

### DISCUSSION THREAD

**Discussion by jb-ashish** [oguimbal/pgsql-ast-parser#78] [Q&A] [answered]
'select order, name from ts_users WHERE name = "ash"';

Error: Syntax error at line 1 col 13:

  select order,
              ^
Unexpected comma token: ",". Instead, I was expecting to see one of the following:

**Comment by oguimbal** [comment, marked as answer]
see my remark on your issue: `order` is a keyword, it cannot be used as a name without double-quoting it `"order"`
